recipe/meta.yaml

```diff
@@ -19,21 +19,21 @@ source:
     - h5py_win_setup_build.py.patch
 
 build:
-  number: 2
+  number: 3
 
 requirements:
   build:
     - python
     - numpy x.x
-    - hdf5
+    - hdf5 1.8*
     - cython
     - six
     - pkgconfig
 
   run:
     - python
     - numpy x.x
-    - hdf5
+    - hdf5 1.8*
```

Comment:
Yeah, do we want to go that route? I mean this has caused you guys so many issues I wonder if it wouldn't be better to ping this to `1.8.15.1` until we get the rest sorted.